### PR TITLE
Log messages: online deletion

### PR DESCRIPTION
**Pull request by mDuo13**
Documents the online deletion log messages upgraded to warnings as of rippled 1.5.0 ( https://github.com/ripple/rippled/pull/3231/ )

### CODE REVIEW

**Review by mtrippled — APPROVED** [review]
:+1:

File: content/tutorials/manage-the-rippled-server/troubleshooting/understanding-log-messages.md [hunk 1]
```diff
@@ -145,6 +145,19 @@ Messages such as the following are caused by a bug in `rippled` v1.1.0 and earli
...
 
+## Not deleting
+
+Messages such as the following occur when [online deletion is interrupted](online-deletion.html#interrupting-online-deletion):
+
+```text
+SHAMapStore:WRN Not deleting. state: syncing. age 25s
+```
+
+The `state` indicates the [server state](rippled-server-states.html) and the `age` indicates how many seconds since the last validated ledger was closed. (A healthy age for the last validated ledger is 7 seconds or less.)
```
Comment (LoisRP) [inline content/tutorials/manage-the-rippled-server/troubleshooting/understanding-log-messages.md]: The `state` indicates the [server state](rippled-server-states.html), and the `age`

**Review by LoisRP — COMMENTED** [review]
LGTM

**Comment by mDuo13** [comment]
All the broken links are some kind of glitch with the edit links, so this is a pass.

**Pull request merged**